### ISSUE TITLE
Add English "about" link, that links to this GitHub

### ISSUE THREAD

**Issue by jidanni** [der-stefan/OpenTopoMap#247]
When we click on https://www.opentopomap.org/about
there should be an English link,
that links here to
https://github.com/der-stefan/OpenTopoMap !
Good enough for now. No need to translate anything.

(Yes, in the middle of foreign language https://www.opentopomap.org/about one can eventually find a GitHub link.)

**Comment by Juergeen** [comment]
Yes, we should have an English about page. It would be best if the browser language was used and if the browser language is unavailable, the default should be English instead of _weird foreign_ German :-)  
If would do the English translation if this issue is about to be resolved.  
And I would test the result as my browser defaults to English despite I'm German.

**Comment by jidanni** [comment]
I found a way to break through the _obfuscated_ About page to dig out the vital GitHub links that I was sure were lurking inside it:
view-source:https://www.opentopomap.org/about
then CTRL+G github.
